### DISCUSSION TITLE
Possibility of iOS version

### DISCUSSION THREAD

**Discussion by orbitalmartian8** [Floorp-Projects/Floorp#743] [Feature Request]
I mainly use everything on my iPhone and I'd love to have this browser available to use on iOS :) Thanks.

**Comment by CosmicIndustries** [comment]
Android as well:)

**Reply by neurodiverseEsoteric** [reply]
This would only be desired if it were the first firefox-based app to implement containers, in a way that allowed for using the "temporary containers" addon as well...

**Comment by Telmesomething** [comment]
They're both available already. To find them, click the shield to the left of the URL bar and then click the link to "Protections dashboard". Scroll down to "Block ad trackers across more devices" and you'll find the link to both of them in there.

**Reply by CosmicIndustries** [reply]
Whut???

**Reply by CosmicIndustries** [reply]
> Whut???

I dont see anything you are talking about

**Reply by Telmesomething** [reply]
Well, it clearly says that there's a Floorp browser for Android and IOS so I assumed that was the case. Maybe it's something which is being planned, but not released yet.

**Comment by Telmesomething** [comment]
Partial correct answer to your question concerning Android at least: https://support.ablaze.one/articles/floorp/floorp-android

**Reply by CosmicIndustries** [reply]
When is it planned to release the Android version? 
Do you have plans to release Floorp Android version? 
Last modified date: 2023-03-12
 ANDROID version will start developing this year this year, but since there is no developer who can write languages such as KOTLIN, the release period is undecided.

**Comment by CutterKnife** [comment]
Not planned at this time due to lack of resources.

**Comment by Coool** [comment]
You need to change title to "Possibility of iOS and Android version"
